infra/app/service/main.tf

```diff
@@ -70,6 +71,13 @@ data "aws_iam_policy" "db_access_policy" {
   name  = local.database_config.access_policy_name
 }
 
+# Retrieve url for external incident management tool (e.g. Pagerduty, Splunk-On-Call)
+
+data "aws_ssm_parameter" "integration_url" {
```

Comment:
need better name than integration url. integration url for what?

Reply:
let it be 
incident_management_service_integration_url